protocols/kad/src/behaviour.rs

```diff
@@ -778,7 +778,7 @@ where
         self.kbuckets
             .closest(key)
             .filter(move |e| e.node.key.preimage() != source)
-            .take(self.queries.config().replication_factor.get())
+            .take(K_VALUE.get())
```

Comment:
This is the number of closer peers that the server will return upon receiving a request. Using `K_VALUE` is correct